### ISSUE TITLE
Watch for Tesla not working

### ISSUE THREAD

**Issue by SW00000** [Kemmey/Kemmey-TeslaWatch-Public#144]
Since getting Apple Watch series 6 on Friday Watch for Tesla app not working properly.  First I could no longe use remote start feature of this app.  After deleting and reinstalling the app, now I can not even log into the app any longer!  What is the problem and is anyone else having the same issues?

**Comment by Kemmey** [comment]
@SW00000 How are you logging in and what errors do you see?

I've added some more debug output when logging in, in the latest beta. Could you install that?

https://testflight.apple.com/join/FdzFOLZv

**Comment by SW00000** [comment]
I do not want to put beta software on my brand new Apple Watch Series 6.
I paid for this app and now it does not work with the most important feature of the app.
I really want this to work again as it did before so let me know when you get this fixed and I will then try it once again. Until then, I will not be using your app.

**Comment by Kemmey** [comment]
That's entirely fair, but if you will not work with me to find a solution to the problem, you will have to wait until someone else runs into the same problem and is willing to work with me. I can't solve a problem I don't know about.

**Comment by Kemmey** [comment]
And just to be clear - the beta version of the app will be much more stable and robust than the current shipping release - that's just the nature of software development. The AppStore release has known bugs. The TestFlight version has fewer, and thus, is more stable and better :-D

**Comment by SW00000** [comment]
Please stay in touch so I know when this is working again.  I just do not want to take a chance on harming my Watch by using Beta software.  I can not be the only person with this problem so I am hopeful you will get back in touch with me.  thanks.

**Comment by Kemmey** [comment]
I just tried remote start on my watch with watchOS 7 - works perfectly fine. 

Did you toggle it off and back on again and re-enter your password?

**Comment by SW00000** [comment]
One more thing, I may change my mind so I will be in touch if I do so.  I am just a little uncomfortable doing this without support from Apple. Do you provide phone support if I need help with the download process?

**Comment by Kemmey** [comment]
Oh - and did you manage to log back in at all?

**Comment by SW00000** [comment]
Yes, but I will do it again now.  Hang on.

**Comment by SW00000** [comment]
I am getting error message when I try to log back in.  I am unable to even log in anymore.

**Comment by SW00000** [comment]
Just deleted app and re downloading it now and will try again to log in.

**Comment by SW00000** [comment]
No luck.  Still getting error message upon trying to log in.
It says “error occurred”.

**Comment by Kemmey** [comment]
Before you log back in, please log in to the Tesla.com website to ensure your account is not locked out from too many failed attempts. They lock it after two failed attempts with bad password. Just as a precaution.

**Comment by Kemmey** [comment]
Ok. I'm planning to ship the watchOS 7 update this week, so if you can wait, that's the time frame.

**Comment by SW00000** [comment]
Yes, I am now locked out and will need to call them.  I can not log in even by changing my password which is what they suggest.  Once i am able to get back in I will contact you.

**Comment by SW00000** [comment]
I will wait.  Please let me know when you ship I can be sure to get the update.
Thanks.

**Comment by Kemmey** [comment]
What?! Completely absolutely logged out?!? That seems harsh - I've never heard of that happening.

**Comment by SW00000** [comment]
Just got off the phone with Tesla and they were able to reset my password.
I will now wait for your update.
Please let me know when it is available.
thanks.

**Comment by Kemmey** [comment]
Fantastic - glad to hear :-D 

Honestly, you don't have to wait for an update. I believe the problem has been your locked account all along - it might have been locked out initially when you transitioned to your new watch. 

Please try to login just once with your new password - and make triple sure you get it right - both when logging into the app - and when setting it for remote start. 

The update will only show more detailed error messages - it won't change anything about the functionality of logging in. 

But that is entirely up to you :-D

**Comment by SW00000** [comment]
Just tried it and I think all is working now. Will confirm later as I am not near my Tesla right now. Thanks.

**Comment by Kemmey** [comment]
1.0.7 is now live in the AppStore - should improve the experience on watchOS 7 :-D

**Comment by SW00000** [comment]
Now remote start does not work. What do I need to do to activate it beyond logging in?  Can I move it up also so I see it as soon as I open the app?  Thanks.

**Comment by Kemmey** [comment]
You need to activate it in settings. It requires your account password to work. So you need to turn it on in settings and input your account password. Please ensure you triple check your password, as your account will get locked out again if there are typos.

**Comment by Kemmey** [comment]
You can reorder the controls from settings as well and put it on the main screen. 

Note that once enabled, you can also have it auto-enable when you tap the unlock button. Then it will unlock your charge port, unlock your car and start remote start just with one tap on unlock button...

**Comment by SW00000** [comment]
Sounds good. Let’s see if it works later when I get back to my car.

**Comment by SW00000** [comment]
Done that. Will try again in few hours and get back to you. Thanks.

**Comment by SW00000** [comment]
I believe Tesla Mobile app and on line for signing on is down. Can not connect to anything from internet except their main page. Can you tell me if you are having same experience.

**Comment by SW00000** [comment]
Working again. They were down updating web pages.

**Comment by Kemmey** [comment]
Global issue apparently 

https://electrek.co/2020/09/23/tesla-suffers-complete-network-outage-internal-systems-and-connectivity-features-down/

**Comment by SW00000** [comment]
Thanks. Seems to be working again for me.

**Comment by SW00000** [comment]
So, I accidentally erased my Watch and now in restoring it to a backup I can not get your App to connect to my vehicle.  I assume that is due to the outage which must be still going on.  Is that your understanding?  Thanks for all of your help.

**Comment by Kemmey** [comment]
Yeah, it's quite spotty right now. Right now I can connect to my vehicle, but it's a bit irratic. It's completely down in the US still, from what I hear, so it will vary across the globe.

**Comment by SW00000** [comment]
Thanks.  Let’s see how long this lasts.  Thanks.
